### DISCUSSION TITLE
胸腔X光結合AI揪出早期肺癌 明年結合健保系統上線

### DISCUSSION THREAD

**Discussion by jacky0115** [1101-GS3073-C/Python-Community-GS3073-C#187] [科技新知]
[胸腔X光結合AI揪出早期肺癌 明年結合健保系統上線](https://health.ltn.com.tw/article/breakingnews/3777888)
此套AI模組可以輔助醫生判斷胸腔是否有異常，大量縮減原本往往需要數周等待結果出爐的時間，有了這項AI模組就能夠及時「示警」胸腔異常，讓醫師及早介入治療。

**Comment by briankuo123** [comment]
能夠大幅縮短判斷結果的時間真的是AI一個很厲害的優勢。

**Comment by minniewei** [comment]
希望這項產品能在家戶間普及!!!!這樣就能救更多的人!!!

**Comment by ghost** [comment]
很期待這個AI能夠快速的普及社會 是非常棒的構思

**Comment by jim11261314** [comment]
我認為AI在檢測技術上的應用，確實有極大的潛力
以往人工辨別容易出錯，且會耗費大量時間

**Comment by LanningYang** [comment]
AI的數據能提高醫師的病情判斷，減少錯誤的判斷
